src/modules/user/components/user-contact-popup/user-contact-popup.tsx

```diff
@@ -8,33 +8,64 @@ import type { UserContactPopupProps } from './user-contact-popup.interface';
 import styles from './user-contact-popup.module.scss';
 
 export const UserContactPopup: FC<UserContactPopupProps> = ({ ...props }) => {
+  const professional = props.professional;
```

Comment:
You can use `export const UserContactPopup: FC<UserContactPopupProps> = ({ professional, ...props }) => {...}` instead of creating additional const.